media/src/components/chat/ChatRow.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Flex, Heading, VStack, Text, useColorMode } from '@chakra-ui/react'
+
+
+const ChatRow = () => {
```

Comment:
ts-docs would really pull this together! as well as adding props (even if the parent passes in dummy content) as that would allow me to understand from a review perspective what the intent is behind this component.